fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -23,95 +24,118 @@ import STRINGS from 'resources/strings';
 
 import { EvotingHooks } from '../hooks';
 import { requestCreateElection } from '../network/ElectionMessageApi';
-import { Question } from '../objects';
+import { ElectionVersion, Question } from '../objects';
 
 const DEFAULT_ELECTION_DURATION = 3600;
 
 // for now only plurality voting is supported (2022-03-16, Tyratox)
 const VOTING_METHOD = STRINGS.election_method_Plurality;
 
+// the type used for storing questions in the react state
+// does not yet contain the id of the questions, this is computed
+// only on creation of the election
+// ALSO: for now the write_in feature is disabled (2022-03-16, Tyratox)
+type NewQuestion = Omit<Question, 'id' | 'write_in'>;
```

Comment:
if the `write_in` is disabled, shouldn't we remove it altogether from `Question` for the time being ? 
It might be more straightforward than having this kind of annotations here and there in the codebase.

Reply:
Yes I agree but this would require the protocol to change as well and I thought for the moment it might be easier to leave the field in the protocol and just fix its value to `false`?

Reply:
good point, I hadn't thought about the protocol implications.